config/software/ruby-cleanup.rb

```diff
@@ -0,0 +1,175 @@
+#
+# Copyright:: Copyright (c) 2014-2020, Chef Software Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Common cleanup routines for ruby apps (InSpec, Workstation, Chef, etc)
+#
+require "fileutils"
+
+name "ruby-cleanup"
```

Comment:
Can we just use the version of this config from omnibus-software instead of duplicating it here?

Reply:
The version in omnibus-software, checks for single bundler version, which fails for toolchain as we have no overrides and don't need bundler check here.  So I ported the config here without the bundler check

Reply:
Can you replace the comment in this will with that explaination. It would be nice to know where we sourced this from originally and why we had to fork it.